cloud/misc/rhevm.py

```diff
@@ -0,0 +1,1476 @@
+#!/usr/bin/python
+
+# (c) 2016, Timothy Vandenbrande <timothy.vandenbrande@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible. If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: rhevm
+author: Timothy Vandenbrande
+short_description: RHEV/oVirt automation
+description:
+    - Allows you to create/remove/update or powermanage virtual machines on a RHEV/oVirt platform.
+version_added: "2.2"
+requirements:
+    - ovirtsdk
+options:
+    user:
+        description:
+            - The user to authenticate with.
+        default: "admin@internal"
+        required: false
+    server:
+        description:
+            - The name/ip of your RHEV-m/oVirt instance.
+        default: "127.0.0.1"
+        required: false
+    port:
+        description:
+            - The port on which the API is reacheable.
+        default: "443"
+        required: false
+    insecure_api:
+        description:
+            - A boolean switch to make a secure or insecure connection to the server.
+        default: false
+        required: false
+    cluster:
+        description:
+            - The rhev/ovirt cluster in which you want you VM to start.
+        required: true
+    state:
+        description:
+            - This serves to create/remove/update or powermanage your VM.
+        default: "present"
+        required: false
+        choices: ['present', 'absent', 'up', 'down', 'restart', 'cd']

```

Comment:
+            state      = dict(default='present', choices=['ping', 'present', 'absent', 'up', 'down', 'restarted', 'cd']),

Ping

Your code looks for restarted, rather than restart